_data/guides.yml

```diff
@@ -81,9 +81,9 @@
   - id: docs_cli_logout
     path: /docs/cli/logout
     tags: ["cli-logout"]
-  - id: docs_cli_ls
-    path: /docs/cli/ls
-    tags: ["cli-ls"]
+  - id: docs_cli_list
+    path: /docs/cli/list
+    tags: ["cli-list"]
```

Comment:
This should reordered to be above login. logout in order to keep the commands in alpha order